src/cleed++/Util/teebuf.hh

```
@@ -0,0 +1,56 @@
+
+#include <string>
+#include <istream>
+#include <ostream>
+
+template <typename char_type,
+      typename traits = std::char_traits<char_type> >
+class basic_teebuf:
+  public std::basic_streambuf<char_type, traits>
+{
+public:
+  typedef typename traits::int_type int_type;
```

Clang (14.0.6)

[warning] use 'using' instead of 'typedef'

(modernize-use-using)